### PR TITLE
Touch introduction

### LINKED ISSUE #3189: touch screen introduction
**Describe your proposal**
We should add a introduction how to use the touch screen and it gestures in VCMI.

It often happens that users do not know how to do something (e.g. long touch for right mouse button, attacking from direction, ...).

Because the knowledge is missing, the input is often rated as imprecise.

**Additional context**
Maybe show a window on first touch input.

As far as I know, this is also available in the official HD android version (@dydzio0614 Didn't you post a video of it somewhere? Do you still have it?).

An idea would to add a window with a video showing basic input. But it should be simple ("vector" based, not using any H3 assets). Do we have anyone who is familiar with creating animations?

### PR DESCRIPTION
**Pull request by Laserlicht**
Fixes #3189

Todo:
- [x] Move strings to translation
- [x] Implement Next/Previous
- [x] Embed Video/Image/Animation

Mod for removing copyrighted material before screencast:
[testmod.zip](https://github.com/vcmi/vcmi/files/13394874/testmod.zip)
- Sword is from: https://opengameart.org/content/cc0-sword-icons
- Used town is https://github.com/vcmi-mods/asylum-town/
- Screencast is done via samsung device (which also capture touch input) and cropped and converted over ffmpeg (XMedia Recode)

https://github.com/vcmi/vcmi/assets/13953785/b689202d-b24b-408e-a2fe-b2510db1a8de

### CODE REVIEW

**Comment by Laserlicht** [comment]
@IvanSavenko 850kB should be okay? ;)

File: Mods/vcmi/config/vcmi/english.json [hunk 1]
```diff
@@ -172,6 +172,14 @@
 
 	"vcmi.battleResultsWindow.applyResultsLabel" : "Apply battle result",
 
+	"vcmi.tutorialWindow.title" : "Touchscreen Introduction",
+	"vcmi.tutorialWindow.decription.RightClick" : "Touch and hold the element on which you want to right-click. Touch the free area to close.",
+	"vcmi.tutorialWindow.decription.MapPanning" : "Panning with one finger to move the map.",
+	"vcmi.tutorialWindow.decription.MapZooming" : "Pinching with two fingers to zoom the map. Standard zoom has a snap-in effect.",
+	"vcmi.tutorialWindow.decription.BattleDirection" : "To attack from a defined side, swipe in the corresponding direction from which the attack is to be made.",
+	"vcmi.tutorialWindow.decription.BattleDirectionAbort" : "The attack direction gesture can be canceled if the finger is far enough away.",
+	"vcmi.tutorialWindow.decription.AbortSpell" : "A spell can be canceled by touching and holding.",
```
Comment (Alexander-Wilms) [inline Mods/vcmi/config/vcmi/english.json:181]: Suggestions:

```
	"vcmi.tutorialWindow.decription.MapPanning" : "Touch and drag with one finger to move the map.",
	"vcmi.tutorialWindow.decription.MapZooming" : "Pinch with two fingers to change the map zoom.",
	"vcmi.tutorialWindow.decription.BattleDirection" : "To attack from a particular direction, swipe in the direction from which the attack is to be made.",
	"vcmi.tutorialWindow.decription.BattleDirectionAbort" : "The attack direction gesture can be cancelled if the finger is far enough away.",
	"vcmi.tutorialWindow.decription.AbortSpell" : "Touch and hold to cancel a spell.",
```

**Comment by IvanSavenko** [comment]
Kind of wish we had better UI library - e.g. for buttons that can be generated in runtime from H3 resources...
"Back" / "Next" / "Finish" buttons would be more intuitive here.

**Comment by Laserlicht** [comment]
> Kind of wish we had better UI library - e.g. for buttons that can be generated in runtime from H3 resources... "Back" / "Next" / "Finish" buttons would be more intuitive here.

Yes, would be great. Official HD has "real" left right buttons. But no priority yet.

**Pull request merged**